modules/react/loading-dots/lib/LoadingDots.tsx

```diff
@@ -15,12 +15,13 @@ const keyframesLoading = keyframes({
   },
 });
 
-const singleLoadingDotStencil = createStencil({
+export const singleLoadingDotStencil = createStencil({
```

Comment:
pulling out this comment https://github.com/Workday/canvas-kit/pull/2991#discussion_r1816379374 as I'm still not sure the implementation of the stencils here is quite right:

Does exporting this stencil for consumers to override work if we don't also expose the `LoadingAnimationDot` component? Do overrides on a parent apply to children? (I'm really not familiar with this so please correct ruthlessly 😂 ) 

```tsx
// In consumer
const loadingDotsStyles = createStyles({
  [singleLoadingDotStencil.vars.color]: system.color.icon.inverse,
});

<LoadingDots cs={loadingDotsStyles}

// LoadingDots.tsx implementation
export const LoadingDots = createComponent('div')({
  displayName: 'LoadingDots',
  Component: (elemProps: CSProps, ref, Element) => {
  Component: ({backgroundColor, ...elemProps}: LoadingDotsProps, ref, Element) => {
    return (
      <Element ref={ref} {...handleCsProp(elemProps, loadingDotsStencil())}> // I was assuming `loadingDotsStyles` was being applied here
        <LoadingAnimationDot /> // Does this get the override in `loadingDotsStyles`?
        <LoadingAnimationDot />
        <LoadingAnimationDot />
      </Element>
    );
  },
});
```